src/features/layer_features_panel/atoms/layerFeaturesCollectionAtom.ts

```
@@ -34,24 +49,27 @@
       state = null;
     });
 
-    onChange('focusedGeometryAtom', (fg) => {
+    onChange('focusedGeometryAtom', (focusedGeometry) => {
       // @ts-expect-error needs better atom type
       currentFeatureIdAtom.set.dispatch(undefined);
       state = null;
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)